File: core/src/com/unciv/models/gamebasics/TechColumn.java
package com.unciv.models.gamebasics;

import java.util.ArrayList;

public class TechColumn {
    public int columnNumber;
    public String era;
    public GameBasics gameBasics;
    public ArrayList<Technology> techs = new ArrayList<Technology>();
    public int techCost;
    public int buildingCost;
    public int wonderCost;
}

File: core/src/com/unciv/models/gamebasics/TechColumn.kt
package com.unciv.models.gamebasics

import java.util.*

class TechColumn {
    var columnNumber: Int = 0
    lateinit var era: TechEra
    var techs = ArrayList<Technology>()
    var techCost: Int = 0
    var buildingCost: Int = 0
    var wonderCost: Int = 0
}


enum class TechEra{
    Ancient,
    Classical,
    Medieval,
    Renaissance,
    Industrial,
    Modern,
    Information,
    Future
}